### DISCUSSION TITLE
"bw" not found in path -- but it's there

### DISCUSSION THREAD

**Discussion by mckeehan** [blacs30/bitwarden-alfred-workflow#193] [Q&A]
When I use the alfred workflow, I get an error

```
07:07:40 workflow.go:376: [ERROR] "bw" not found in path "/usr/bin:/usr/local/bin:/usr/local/sbin:/usr/local/share/npm/bin:/usr/bin:/usr/sbin:/opt/homebrew/bin:"
```

```
❯ which bw
/opt/homebrew/bin/bw
```

I am at a loss as to how this should be resolved. Does anyone have any suggestions?

**Comment by luckman212** [comment]
I doubt it's the problem, but 2 things I noticed are

1. `/usr/bin` appears twice in your `PATH`
2. your `PATH` ends with a colon, this is not typical

Maybe try removing those and see if it has any effect?

**Reply by mckeehan** [reply]
Thanks for the suggestion. As you predicted, those changes do not appear to have any impact.

```
[14:39:36.608] Bitwarden v2[Run Script] Passing output '"bw" not found in path "/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/usr/local/share/npm/bin:/opt/homebrew/bin"
```

**Reply by blacs30** [reply]
Have you tried moving the `/opt/homebrew/bin` path first in the PATH environment variables of workflow

**Reply by mckeehan** [reply]
I had not tried that...but I get similar results

```
"bw" not found in path "/opt/homebrew/bin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/usr/local/share/npm/bin"
```
